src/reason_pprint_ast.ml

```diff
@@ -3578,10 +3548,12 @@ class printer  ()= object(self:'self)
   method formatJSXComponent componentName args =
     let rec processArguments arguments processedAttrs children =
       match arguments with
-      | ("", {pexp_desc = Pexp_construct (_, None)}) :: tail ->
+      | ("children", {pexp_desc = Pexp_construct (_, None)}) :: tail ->
```

Comment:
I haven't been following jsx for a while so my understanding maybe lagging, but is this logic going to make the parser locked to reactjs ("children")? Should we somehow move it to reactjs_jsx_ppx?